### ISSUE TITLE
ServerlessError: An error occurred: WebsocketsDeploymentStage

### ISSUE THREAD

**Issue by asafvaron90** [serverless/serverless#7016]
Hi, I'm trying to deploy serverless with websockets and receiving the following error

`ServerlessError: An error occurred: WebsocketsDeploymentStage - The following context variables are not supported: [$context.status] (Service: AmazonApiGatewayV2; Status Code: 400; Error Code: BadRequestException; Request ID: 0722a2e7-7a0b-41d1-8cbf-14ea8767a4f3).`

Thanks for the help

**Comment by pleizi** [comment]
Same here!

An error occurred: WebsocketsDeploymentStage - The following context variables are not supported: [$context.authorizer, $context.error] (Service: AmazonApiGatewayV2; Status Code: 400; Error Code: BadRequestException; Request ID: 8b3b9c94-1f2d-49f7-838b-2f82ab746bcd).

Thanks for the help!!!!

**Comment by Filip-Duc** [comment]
Same here

**Comment by silberistgold** [comment]
Same here. I am using a very simple setup. Config looks something like this:

```

**Comment by silberistgold** [comment]
Same here. I am using a very simple setup. Config looks something like this:

**Comment by silberistgold** [comment]
Same here. I am using a very simple setup. Config looks something like this:

```
functions:
  connectionHandler:
    handler: handler.connectionHandler
    events:
      - websocket:
          route: $connect
      - websocket:
          route: $disconnect

  defaultHandler:
    handler: handler.defaultHandler
    events:
      - websocket: $default
```

**Comment by exoego** [comment]
I think this was resolved in #7014 
Stay tuned for upcoming release.
